Jenkinsfile

```
@@ -22,177 +22,16 @@ pipeline {
-                      -s \
-                      -H 'Metadata-Flavor: Google' \
-                      "http://metadata/computeMetadata/v1/instance/service-accounts/default/identity?format=${token_format}&audience=${audience}"
-                  }
-
-                  echo "$(retrieve_token "full" "conjur/cucumber/host/test-app")" > "gce_token_valid"
-                  echo "$(retrieve_token "full" "conjur/cucumber/host/non-existing")" > "gce_token_non_existing_host"
-                  echo "$(retrieve_token "full" "conjur/cucumber/host/non-rooted/test-app")" > "gce_token_non_rooted_host"
-                  echo "$(retrieve_token "full" "conjur/cucumber/test-app")" > "gce_token_user"
-                  echo "$(retrieve_token "full" "conjur/non-existing/host/test-app")" > "gce_token_non_existing_account"
-                  echo "$(retrieve_token "full" "invalid_audience")" > "gce_token_invalid_audience"
-                  echo "$(retrieve_token "standard" "conjur/cucumber/host/test-app")" > "gce_token_standard_format"
-                '''
-
-                stash name: 'authnGceTokens', includes: 'gce_token_valid,gce_token_invalid_audience,gce_token_standard_format,gce_token_user,gce_token_non_existing_host,gce_token_non_existing_account,gce_token_non_rooted_host', allowEmpty:false
-                env.GCE_TOKENS_FETCHED = "true"
-              }
-            }
-          }
-        }
-        stage('GCE Authenticator') {
-          steps {
-            timeout(time: 10, unit: 'MINUTES') {
-              waitUntil {
-                script {
-                  return (env.GCE_TOKENS_FETCHED == "true")
-                }
-              }
-            }
-            script {
-              dir('ci/authn-gce/tokens') {
-                unstash 'authnGceTokens'
-              }
-
-              sh 'ci/test cucumber_authenticators_gce'
-            }
-          }
-        }
-        stage('Policy') {
-          steps { sh 'ci/test cucumber_policy' }
-        }
-        stage('API') {
-          steps { sh 'ci/test cucumber_api' }
-        }
-        stage('Rotators') {
-          steps { sh 'ci/test cucumber_rotators' }
-        }
-        stage('Kubernetes 1.7 in GKE') {
-          steps { sh 'cd ci/authn-k8s && summon ./test.sh gke' }
-        }
-        stage('Audit') {
-          steps { sh 'ci/test rspec_audit'}
-        }
-      }
-    }
-
-    stage('Submit Coverage Report'){
-      steps{
-        sh 'ci/submit-coverage'
-      }
-    }
-
-    stage('Push Docker image') {
-      steps {
-        sh './push-image.sh'
-      }
-    }
-
-    stage('Build Debian package') {
-      steps {
-        sh './package.sh'
-
-        archiveArtifacts artifacts: '*.deb', fingerprint: true
-      }
-    }
-
-    stage('Publish Debian package'){
-      steps {
-        sh './publish.sh'
-      }
+    stage('Run Tests OCP 4.3') {
+       steps {
+        sh 'cd ci/authn-k8s && summon --environment openshift43 ./test.sh openshift43'
```

Edit line to run tests on X OCP version
```suggestion
        sh 'cd ci/authn-k8s && summon --environment openshiftX ./test.sh openshiftX'
```